apps/studio/src/shared/lib/knex-clickhouse/index.ts

```diff
@@ -0,0 +1,369 @@
+// "forked" from  https://github.com/cleverbrush/framework/blob/e1dbee24d205ddb84807ae5327c8fe9592cd6064/libs/knex-clickhouse/src/index.ts
```

Comment:
This can't be included unless they make it available under the MIT/Apache2 license. Even then you need to replicate the copyright notice at the top.

Otherwise you'll have to clean-room reimplement this file I'm afraid. I filed an issue with them to ask them to include a license.

Reply:
https://github.com/cleverbrush/framework/issues/53

Reply:
Actually, looks like they're using BSD 3-clause. So you'll need a copyright statement on the files you've forked.

Reply:
Seems like we need them to add their license file first so we can copy it over to our forked files?